items.py

```diff
@@ -123,7 +123,7 @@
     }
 
     files['/etc/restic/password_{host_name}'.format(host_name=backup_host)] = {
-        'content': repo.libs.pw.get("restic_password_{host_name}_{node_name}".format(
+        'content': repo.vault.password_for("restic_password_{host_name}_{node_name}".format(
```

Comment:
aehh.. was is denn an der pw.get lib schlecht? und warum vault? was haengt da noch dran?

Reply:
pw.get ist selfmade und vault gehört mittlerweile zu BW dazu, man muss sich also nicht noch die lib mit reinziehen.

Reply:
aber stored auf nem mac den key nicht in der keychain, sondern in einer datei